kobo/apps/project_ownership/docs/api/v2/project_ownership/invites/create.md

```diff
@@ -0,0 +1,5 @@
+## Create (send) an invite
+
+Where the assets `uid` are given in the payload's list
+
```

Comment:
I think something is configured wrong because every time I send a POST request I get this error. When I try the exact same request locally, it works fine. 

<img width="1104" height="701" alt="Image" src="https://github.com/user-attachments/assets/a2baa81a-b0c0-4690-8d21-e37a2efba2e4" />